src/FixedDataTableCellGroup.js

```diff
@@ -79,7 +84,7 @@ class FixedDataTableCellGroupImpl extends React.Component {
   render() /*object*/ {
     const props = this.props;
     const columns = props.columns;
-    const cellGroupWidth = sumPropWidths(columns);
```

Comment:
Ran through Chrome's profiler and saw that this single line causes a big performance drop when there exists a huge no:of scrollable columns. This is because we're calculating the total width of millions of columns every render per cell group.

Moved this over to the reducer, where it only get's calculated ONCE, and then passed down to the components.

Notice that we'll still calculate for fixed columns since they aren't virtualized.